__tests__/app.test.js

```diff
@@ -50,14 +50,87 @@ describe('4: /api/reviews', () => {
                 });
             });
     });
-    test('GET: 200 - produces a reviews array sorted by date in descending order', () => {
+    test('GET: 200 - produces a reviews array sorted by date in descending order (by default when no query given)', () => {
         return request(app)
             .get('/api/reviews')
             .expect(200)
             .then(({body}) => {
                 expect(body.reviews).toBeSortedBy('created_at', {descending: true});
             });
     });
+    describe('11: /api/reviews queries', () => {
+        test('GET: 200 - when given category query returns reviews array only from that category', () => {
+            return request(app)
+                .get('/api/reviews?category=social deduction')
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.reviews.length).toBe(11)
+                    body.reviews.forEach((review) => {
+                        expect(review.category).toBe('social deduction');
+                    });
+                });
```

Comment:
Another test could also be handy. What if the category is valid but has no reviews associated with it? This should be a 200 with an empty array returned 🙂